### PR TITLE
Retry push errors

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: bin/push.sh [hunk 1]
```diff
@@ -3,6 +3,9 @@
 set -xeuo pipefail
 
 while [ "$#" -gt 0 ]; do
-	docker push "$1"
-	shift
+    while ! docker push "$1"; do
+	    echo "Failed to push $1, retrying in 30s..."
+	    sleep 30
```
Comment (kokosing) [inline bin/push.sh]: no timeout?

Comment (findepi) [reply bin/push.sh]: do i need timeout?

`make release` takes hours anyway

Comment (kokosing) [reply bin/push.sh]: so you could just limit the number of attempts.

Comment (findepi) [reply bin/push.sh]: Currently, this is always run "interactively" (manually) and in the background (it takes very long). If I leave the release running unattended, I want it to do as much as possible to complete, before I come to check the results. I would not want to limit number of attempts unless there is some downside to not doing so.

**Pull request merged**